app/models/collection.rb

```diff
@@ -116,36 +114,36 @@ def parent_is_allowed
       if parent == self
         errors.add(:base, ts("You can't make a collection its own parent."))
       elsif parent_id_changed? && !parent.user_is_maintainer?(User.current_user)
-        errors.add(:base, ts("You have to be a maintainer of %{name} to make a subcollection.", :name => parent.name))
+        errors.add(:base, ts("You have to be a maintainer of %{name} to make a subcollection.", name: parent.name))
       end
     end
   end
 
-  validates_presence_of :name, :message => ts("Please enter a name for your collection.")
-  validates_uniqueness_of :name, :case_sensitive => false, :message => ts('Sorry, that name is already taken. Try again, please!')
+  validates_presence_of :name, message: ts("Please enter a name for your collection.")
+  validates_uniqueness_of :name, case_sensitive: false, message: ts('Sorry, that name is already taken. Try again, please!')
   validates_length_of :name,
-    :minimum => ArchiveConfig.TITLE_MIN,
-    :too_short=> ts("must be at least %{min} characters long.", :min => ArchiveConfig.TITLE_MIN)
+    minimum: ArchiveConfig.TITLE_MIN,
+    too_shor: ts("must be at least %{min} characters long.", min: ArchiveConfig.TITLE_MIN)
```

Comment:
Just wanted to point out that there are a couple places in here where it got changed from `too_short` to `too_shor` without the t, or from `too_long` to `too_lon` with the g missing.